### DISCUSSION TITLE
When I push to Git, the 'spacebar-website' is automatically being built.

### DISCUSSION THREAD

**Discussion by ao-xyz** [vercel/community#2504] [Help]
This folder is not included in my current project settings.
I suspect this is either a project that has already been deleted, or it's linked to the account of an employee who has left the company.
I would like to prevent it from being built.

https://vercel.com/ao-labs-outdated/spacebar-website/5QeKuyMaSAk6LipNGWAHZn1itVF8

**Comment by amyegan** [comment]
It looks like AO Labs OutDated is a team on an expired Pro trial. I see two projects associated with the team. Both seem to point to the same repo as far as I can tell. If you would like to stop deploying one of these sites, you should delete the associated project and remove the former employee's branch from your git repo.
